### ISSUE TITLE
Feature Request:  Code snippet generation for R

### ISSUE THREAD

**Issue by astewart-twist** [a85/POSTMan-Chrome-Extension#3893]
Code snippet generation in Postman is such a handy feature!  

I'd like to submit a feature request for supporting snippets in R as well.  `httr` (https://github.com/r-lib/httr) is probably the most analogous library to something like python's `requests`.

**Comment by deepakpathania-zz** [comment]
Hey @astewart-twist . Postman actually uses a third party [library](https://github.com/Kong/httpsnippet) to generate code snippets and therefore do not control the languages covered directly. 
I can see that there is already a feature [request](https://github.com/Kong/httpsnippet/issues/101) for R there in the issues, you can probably upvote there to get this some attention.

**Comment by kamalaknn** [comment]
Hi, 

I'm marking this as a feature request. We'll need support from https://github.com/Kong/httpsnippet before we can support it in Postman. Please follow the parent thread for updates.

**Comment by wilder** [comment]
I would like to work on this! How can I proceed? If I want to start working on it which repository should I clone https://github.com/postmanlabs/httpsnippet-fsless or https://github.com/Kong/httpsnippet?

**Comment by a85** [comment]
@WilderPereira I believe Kong's library.

**Comment by wilder** [comment]
@a85 Thanks! I was looking the commits history now, it seems that Kong's library haven't been updated in a while...

**Comment by chakpongchung** [comment]
@astewart-twist @a85 @kamalaknn @wilder @deepakpathania 

Good news! the long awaited R http request support is merged into master branch of 3rd party libraries postman is using: https://github.com/Kong/httpsnippet/pull/131

When should  we expect it to be adapted by postman? coming release?

If I want to manually integrate this into postman by manually building latest postman with this R code generator, how can I build the postman from source code?

**Comment by ismayc** [comment]
Just pinging here again as there are other R users that would like this to be integrated. Thanks in advance!

**Comment by SantoshSrinivas79** [comment]
Any update on this?

**Comment by fljoly** [comment]
Same here, it would be great

**Comment by umeshp7** [comment]
Thanks for the feedback and this feature request. I am happy to let you know that we will be soon adding support for this.

Please let us know your preference on httr vs rcurl. We have started working on both. And it would be great to know which one is more popular or commonly used in the industry. 

You can follow the progress here: https://github.com/postmanlabs/postman-code-generators/issues/347.
